### PR TITLE
Customization changes

### PR DESCRIPTION
**Pull request by franck-boullier**
This still includes the demo users but with additional parameters and permissions.
We use the DB for the BZ FE v2.4.

### CODE REVIEW

File: custom/global/header.html.tmpl [hunk 1]
```diff
@@ -215,7 +215,11 @@
     [%# Required for the 'Autodiscovery' feature in Firefox 2 and IE 7. %]
     <link rel="search" type="application/opensearchdescription+xml"
                        title="[% terms.Bugzilla %]" href="./search_plugin.cgi">
-    <link rel="shortcut icon" href="images/favicon.ico" >
+    <link rel="shortcut icon" href="https://unee-t.com/favicon/favicon.ico" >
+	<link rel="icon" href="https://unee-t.com/favicon/cropped-favicon-32x32.png" sizes="32x32"/>
```
Comment (kaihendry) [inline custom/global/header.html.tmpl:219]: 404s? Guess you are fixing that?

Comment (franck-boullier) [reply custom/global/header.html.tmpl:219]: sorry...
fixed now

**Comment by kaihendry** [comment]
The build isn't working for me. Getting `bugzilla_1  | Not a reference at Bugzilla/DB/Schema.pm line 2916.`.

**Comment by franck-boullier** [comment]
"The build isn't working for me"
@kaihendry is it working now? I see you merged already...

**Comment by kaihendry** [comment]
I am referring to the merged changes aren't working.

**Comment by franck-boullier** [comment]
"I am referring to the merged changes aren't working."
OK weird...

I don't know how dockers works exactly but we might have to do the Bugzilla thing in several different steps:
1- install a vanilla BZ 5.0.3
Once BZ 5.0.3 is up and running
2- Install the custom skin and custom folder
3- run a dedicated script to update the BZ database (the BZ FE and the Bugzilla vanilla database are slightly different)...

Since the BZ and the BZFE databases have different schema this might be why the changes are causing the error...

**Comment by kaihendry** [comment]
I think once I've split up the Docker files, it will be easier to see what's going wrong and iterate.

There is some schema issue, that's for sure.

**Comment by franck-boullier** [comment]
One test we might want to do AFTER we have updated the BZ database (step 3 described above) is to then run the checksetup.pl script once more: it might reset a few things in the database but it should not complain...
That should help us verify if there is an error in our BZ schema update.

Context: in BZ, there is a table that documents the changes done in the BZ vanilla database if you do some customization in the BZ back end.
This is so that when you create some new custom fields, BZ knows about them and behaves nicely if we have to run the checksetup.pl script again, for a version upgrade for instance.

**Pull request merged**